docs/features/authentication.md

```diff
@@ -0,0 +1,15 @@
+# Authentication
+
+The authentication is done via a SSO system.
+
+The general flow for the authentication is described below:
```

Comment:
Preview for the diagram: https://github.com/rezoleo/lea5/blob/auth-sso/docs/features/authentication.md